docs/rfcs/XeGPU.md

```diff
@@ -286,7 +245,236 @@ In case that certain Xe GPU target does not support atomic operation for a certa
 Attribute `scope` describes the scope of fence. "workgroup" means that the scope is within each work group. "gpu" means the scope is across work groups within the gpu.
 Attribute `Memory_kind` describes the memory kind. "global" means the global memory, "shared" means the shared local memory.
 
-`nbarrier` and `fence` operations lower to uniform instructions, so there is no need to specify the sg_map or VC mode.
+`nbarrier` and `fence` operations lower to uniform instructions, so there is no need to specify the `sg_map`.
+
+## XeGPU Attributes to support Work Item Level semantics  
+
+Attribute `xegpu.sg_map` describes the mapping between work item (WI) WI threads and the 2D tensor specified by the tensor descriptor. To distribute the XeGPU operation to WI threads, the tensor_desc must be specified with the `sg_map` attribute at the tensor description creation time.
+
+Within the sg_map, `wi_layout` specifies the layout of WI threads, describing the mapping of WI threads to the tensor. wi_layout[0] x wi_layout[1] must be equal to the total number of WI threads within a subgroup. `wi_data` specifies the data elements assigned to each WI thread for a single distribution. In the example below, the subgroup has 16 WI threads in wi_layout=[1, 16], each accessing 1 element as specified by wi_data=[1,1]. 
```

Comment:
TensorDesc can cover 1d and 2d. Potentially it can cover more than 2d, but it only focus on 1d and 2d. 

wi_layout and wi_data is always 2d. There is no intension to cover even higher dimension.